tests/Cms/Models/ModelWithContentTest.php

```diff
@@ -443,4 +444,26 @@ public function testUuid()
 		$model = new Page(['slug' => 'foo']);
 		$this->assertInstanceOf(PageUuid::class, $model->uuid());
 	}
+
+	public function testVersion()
+	{
+		$model = new Site();
+		$this->assertInstanceOf(Version::class, $model->version('published'));
+		$this->assertSame('published', $model->version()->id()->value());
+
+		$model = new Page(['slug' => 'foo']);
+		$this->assertInstanceOf(Version::class, $model->version('published'));
+		$this->assertSame('published', $model->version()->id()->value());
```

Comment:
`Todo`: Since the fallback is tested in a separate test:

```suggestion
		$this->assertSame('published', $model->version('published')->id()->value());

		$model = new Page(['slug' => 'foo']);
		$this->assertInstanceOf(Version::class, $model->version('published'));
		$this->assertSame('published', $model->version('published')->id()->value());
```

We may also want to test passing a `VersionId` instance.